src/resources/Organizations/OrganizationInterfaces.ts

```diff
@@ -16,22 +16,31 @@ export interface OrganizationsStatusModel {
     supportActivated: boolean;
 }
 
-export interface OrganizationModel {
+export type OrganizationModel = {
```

Comment:
Whats the goal of converting the interface to a type?

Reply:
I like to keep "extends" for actual extensions, this didn't look right: `export interface OrganizationModel extends AdditionalOrganizationFieldsModel`.

I use `type` when I want to compose interfaces, which was my intent here.

But I guess I could merge the `AdditionalOrganizationFieldsModel` fields back directly

Reply:
I agree with you on this one, it doesn't really extend and its more of a composition. However I do think that the composition is unnecessary for now at least and that the 2 fields should be moved back into the OrganizationModel